modules/services/dunst.nix

```
@@ -172,7 +172,7 @@ in {
 
         mkPath = { basePath, theme, category }:
           "${basePath}/share/icons/${theme.name}/${theme.size}/${category}";
-      in concatMapStringsSep ":" mkPath (cartesianProductOfSets {
+      in concatMapStringsSep ":" mkPath ((if lib.versionAtLeast config.home.stateVersion "24.05" then cartesianProduct else cartesianProductOfSets) {
```

We only target Nixpkgs unstable branch inside the HM master branch so you can simplify this to
```suggestion
      in concatMapStringsSep ":" mkPath (cartesianProduct {
```